options/options.css

```diff
@@ -107,8 +103,8 @@ body {
     font: 15px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Should all fonts be in speech marks for standardisation reasons (besides `serif`, `sans-serif` and `cursive`) or should we only do it if there's a space in the font name like here?